### ISSUE TITLE
[BUG] Update certs for ui chassis

### ISSUE THREAD

**Issue by davidradl** [odpi/egeria#6351]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I notice on the latest code we have different p12 files for the ui chassis and the server chassis. I assume that the server chassis is correct and the ui chassis is expired.
```
   davidradley@admins-MBP egeria % ls -l open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/resources/keystore.p12
-rw-r--r--  1 davidradley  staff  6084 25 Mar 13:45 open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/resources/keystore.p12
```

### Expected Behavior

use the same p12 file for the ui chassis and the server chassis. This will be the same one as the react ui will use. 

### Steps To Reproduce

as above

### Environment

```markdown
- Egeria:
- OS:
- Java:
- Browser (for UI issues):
- Additional connectors and integration:
```

### Any Further Information?

maybe there is a reason for these to be different - in which case which should the react UI use?

**Comment by davidradl** [comment]
@planetf1 I assume we would want this fixed in 3.7- if there is something to fix?

**Comment by davidradl** [comment]
It looks like you generated these as different files - but I cannot see mention of the ones to use for the ui chassis in [https://egeria-project.org/guides/admin/omag-server-platform-transport-level-security/?h=certificates#certificates-for-the-omag-server-platform ](https://egeria-project.org/guides/admin/omag-server-platform-transport-level-security/?h=certificates#certificates-for-the-omag-server-platform )

**Comment by lpalashevski** [comment]
> The egeria-ui was already making use of certs - so the only change made when I added the ssl docs & cert scripts was to ensure the files were copied to the existing locations. this is done in https://github.com/odpi/egeria/blob/master/open-metadata-resources/open-metadata-deployment/certificates/gensamplecerts.sh
> 
> However no changes beyond that were done, nor any testing of the egeria-ui.
> 
> I agree, whatever approach is taken needs also to be documented, but this info needs to come from the UI team initially.
> 
> Can you cover this for now @lpalashevski ?

Yes, I will do that. Lets keep this open until I come back with more details.

**Comment by planetf1** [comment]
this has been done, albeit in a hardcoded way in the charts - other issues capture changes needed to cert handling generally
